dockerfile/anaconda-iso-creator-webui/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Dockerfile to build self-contained boot.iso with Anaconda Web UI.
```

Comment:
This new Dockerfile is a duplication of the existing one. We don't need that. Instead we should look on the script inside which you want to change.

There are two possibilities:
- adjust the script to be more universal
- add a new script which can be called instead of the original one by `--entrypoint`